tests/Tests/500__jupyterhub/pvc-change_verification.robot

```diff
@@ -0,0 +1,76 @@
+*** Settings ***
+Documentation    PVC_CHANGE_VERIFICATION
+...                 Verify  if we can modify default PVC size
+...
+...
+...                 = Variables =
+...                 | Namespace | Required |    Jupyterhub configuration namespace|
+...                 | S_size    | Required |    Supported size |
+...                 | Ns_size   | Required  |   Not Supported PVC size |
+...                 | Size_code | Required |    Python code to check size in notebook |
+#
+Library         SeleniumLibrary
+Library         JupyterLibrary
+Resource         ../../Resources/ODS.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+
+Test Teardown   PVC Size Test Teardown
+
+
+*** Variables ***
+${NAMESPACE}    redhat-ods-applications
+${S_SIZE}       15
+${SIZE_CODE}    import subprocess;
+...    int(subprocess.check_output(['df','-h', '/opt/app-root/src']).split()[8].decode('utf-8')[:-1])
+@{NS_SIZE}      0    -15    6.2    abc
+
+
+*** Test Cases ***
+Verify User Can Spawn Notebook With PVC Change
+    [Documentation]   Verify if user can spawn notebook
+    ...    for supported PVC size git change
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-1228    ODS-1221
+    ...       Resources-PVC
+    Change PVC And Rollout JH Deployment Config    ${S_SIZE}Gi
+    Run Keyword And Continue On Failure   Verify Notebook Size     600s    ${S_SIZE}
+    ${pvc_size}   Get Notebook PVC Size        username=${TEST_USER.USERNAME}   namespace=rhods-notebooks
+    Verify PVC Size     ${S_SIZE}       ${pvc_size}
```

Comment:
just a suggestion for an alternative name `PVC Size Should be`? not asking for the change, if you prefer `Verify PVC Size` pls leave it as it is